README.md

```diff
@@ -2,4 +2,57 @@
 
 LSIF indexer for TypeScript and JavaScript.
 
+## Quick start
+
+### Installation
+
+```sh
+npm install -g @sourcegraph/lsif-typescript
+```
+
+### Indexing a TypeScript project
+
+Navigate to the project root, containing `tsconfig.json`.
+
+```sh
+lsif-typescript index
```

Comment:
We should have an `npm install` step here to make sure that dependencies are also indexed.